src/vs/workbench/browser/parts/editor/tabsTitleControl.ts

```diff
@@ -385,6 +385,7 @@ export class TabsTitleControl extends TitleControl {
 			oldOptions.labelFormat !== newOptions.labelFormat ||
 			oldOptions.tabCloseButton !== newOptions.tabCloseButton ||
 			oldOptions.tabSizing !== newOptions.tabSizing ||
+			oldOptions.tabClosingOrder !== newOptions.tabClosingOrder ||
```

Comment:
@SimonEggert all the changes in this file seem unneeded. We neither need to redraw the tab nor change classes when this setting changes, no?

Reply:
You are right, my bad!